Makefile

```
@@ -10,6 +10,9 @@ TEST_CI_ARGS ?=
 STAGINGSERVER ?= node-www
 LOGLEVEL ?= silent
 OSTYPE := $(shell uname -s | tr '[:upper:]' '[:lower:]')
+ifeq ($(findstring OS/390,$OSTYPE),OS/390)
+OSTYPE ?= OS390
+endif
```

OSTYPE here will be all small-caps following the `tr` above.
```suggestion
ifeq ($(findstring os/390,$OSTYPE),os/390)
OSTYPE ?= os390
endif
```